.github/workflows/call-build-macos.yaml

```
@@ -76,8 +76,10 @@ jobs:
         config:
           - name: "Apple Silicon macOS runner"
             runner: macos-14
+            cmake_version: "3.31.6"
```

[3.31.7](https://github.com/Kitware/CMake/releases/tag/v3.31.7) version was released 2 days ago.

```suggestion
            cmake_version: "3.31.7"
```